radix-engine/src/blueprints/epoch_manager/epoch_manager.rs

```diff
@@ -293,15 +398,34 @@ impl EpochManagerBlueprint {
         let next_validator_set: BTreeMap<ComponentAddress, Validator> =
             validators.into_iter().collect();
 
-        let handle = api.actor_lock_field(
+        let validator_set_handle = api.actor_lock_field(
             OBJECT_HANDLE_SELF,
             EpochManagerField::CurrentValidatorSet.into(),
             LockFlags::MUTABLE,
         )?;
-        let mut validator_set: CurrentValidatorSetSubstate = api.field_lock_read_typed(handle)?;
```

Comment:
_(a note from the author)_

I noticed that this read was not needed, so I deleted it.

But my real question is: do we have some "just overwrite this field with this new value" API operation? (that could avoid some lock costs OR at least turn this into a 1-liner?)

Reply:
Agreed this would be a nice API but short answer, no we don't have this operation yet.